product_docs/docs/pgpool/4.2/pgpool_rel_notes/index.mdx

```diff
@@ -0,0 +1,16 @@
+---
+title: "Release Notes"
+redirects:
+  - ../01_whats_new/
+---
+
+EDB Pgpool-II is enhanced to support additional EDB Postgres Advanced Server capabilities and can be used with PostgreSQL or EDB Postgres Advanced Server.
+
+The Pgpool-II documentation describes the latest version of Pgpool-II including minor releases and patches. The release notes in this section provide information on what was new in each release. For new functionality introduced in a minor or patch release, there are also indicators within the content about what release introduced the feature.
+
+| Version                   | Release Date | Upstream Merges                                                                                                                                                                                                                  | 
+| ------------------------- | ------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | 
+| [4.2.6](09_426_rel_notes) | 2021 Dec 01  | Upstream [4.2.6](https://www.pgpool.net/docs/42/en/html/release-4-2-6.html#bug-fixes)                                                                                                                                     |
+| [4.2.5](10_425_rel_notes) | 2021 Jun 10  | Upstream [4.2.5](https://www.pgpool.net/docs/42/en/html/release-4-2-5.html) and [4.2.4](https://www.pgpool.net/docs/42/en/html/release-4-2-4.html)                                                                                  |
+
+Often only select issues are included in the upstream merges. The specific issues included in the merges are listed in the release note topics.
```

Comment:
Is this true? I know it was for PEM, but is it also true for pgpool?

Reply:
removed this line as per our dicussion